cli.mjs

```diff
@@ -1,4 +1,4 @@
-import { checkbox, input, select } from "@inquirer/prompts";
+import { checkbox, input, select, confirm } from "@inquirer/prompts";

```

Comment:
I don't expect you to change anything, but I wanted to let you know the new hotness for cli development is [commander](https://www.npmjs.com/package/commander)

https://npmtrends.com/commander-vs-inquirer

Reply:
huh, interesting! Let me know how it works!